src/open_producten/producttypen/tests/api/test_prijs.py

```diff
@@ -0,0 +1,432 @@
+                        code="invalid",
+                    ),
+                    ErrorDetail(
+                        string=f"Prijs optie id {non_existing_optie} op index 3 bestaat niet.",
+                        code="invalid",
+                    ),
+                ]
+            },
+        )
+
+    def test_read_prijzen(self):
+        prijs = self._create_prijs()
+
+        response = self.get()
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data["count"], 1)
+        self.assertEqual(response.data["results"], [prijs_to_dict(prijs)])
+
+    def test_read_prijs(self):
+        prijs = self._create_prijs()
+
+        response = self.get(prijs.id)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data, prijs_to_dict(prijs))
+
+    def test_delete_prijs(self):
+        prijs = self._create_prijs()
+        response = self.delete(prijs.id)
+
+        self.assertEqual(response.status_code, 204)
+        self.assertEqual(Prijs.objects.count(), 0)
+        self.assertEqual(PrijsOptie.objects.count(), 0)
+
+    def test_get_actuele_prijs_when_product_type_has_no_prijzen(self):
+        response = self.client.get("/api/v1/producttypen/actuele-prijzen/")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.data,
+            [
+                {
+                    "id": str(self.product_type.id),
+                    "naam": self.product_type.naam,
+                    "upl_naam": self.product_type.uniforme_product_naam.naam,
+                    "upl_uri": self.product_type.uniforme_product_naam.uri,
+                    "actuele_prijs": None,
+                },
+            ],
+        )
+
+    def test_get_actuele_prijzen_when_product_type_only_has_prijs_in_future(self):
+        PrijsFactory.create(
+            product_type=self.product_type, actief_vanaf=datetime.date(2024, 2, 2)
+        )
+
+        response = self.client.get("/api/v1/producttypen/actuele-prijzen/")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.data,
+            [
+                {
+                    "id": str(self.product_type.id),
+                    "naam": self.product_type.naam,
+                    "upl_naam": self.product_type.uniforme_product_naam.naam,
+                    "upl_uri": self.product_type.uniforme_product_naam.uri,
+                    "actuele_prijs": None,
+                },
+            ],
+        )
+
+    def test_get_actuele_prijzen_when_product_type_has_actuele_prijs(self):
+        prijs = PrijsFactory.create(
+            product_type=self.product_type,
+            actief_vanaf=datetime.date(2024, 1, 1),
+        )
+
+        optie = PrijsOptieFactory.create(prijs=prijs)
+
+        response = self.client.get("/api/v1/producttypen/actuele-prijzen/")
```

Comment:
Should use reverse here. 

Also is this not a product type endpoint and should be in the porduct type tests?

Reply:
Also these could be in their own test class